src/components/common/Dialog/Description.tsx

```diff
@@ -0,0 +1,29 @@
+'use client'
+
+import * as DialogPrimitive from '@radix-ui/react-dialog'
+import styled from 'styled-components'
+
+interface StyledDescriptionProps {
+  fontSize?: number
+  fontWeight?: 300 | 400 | 500 | 600 | 700 | 800
+}
```

Comment:
이렇게 관리하면 theme으로 따로 관리하는 이유가 없을 것 같아용
아래처럼 theme을 최대한 응용해도 좋을 것 같은데 어떠신가요 ??

```suggestion

interface StyledDescriptionProps {
  fontSize?: number
  fontWeight?: FontWeight
}

...

const StyledDescription = styled.div<StyledDescriptionProps>`
  font-weight: ${(props) =>
    props.fontWeight ? `${theme.fontWeight[props.fontWeight]}` : 400};
`
```

```tsx
export type FontWeight = 'light' | 'normal' | 'bold' | 'extraBold'
```

Reply:
적용 하겠습니다 !!